pkg/storaged/filesystem/mismounting.jsx

```diff
@@ -51,6 +51,12 @@ export function check_mismounted_fsys(backing_block, content_block, fstab_config
     const other_mounts = mounted_at.filter(m => m != dir);
     const crypto_backing_noauto = get_cryptobacking_noauto(client, backing_block);
 
+    /*
```

Comment:
This comment should go to `get_mount_points`, I'd say.

Reply:
Agreed and a large comment is there so I'll remove it here.